streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java

```diff
@@ -228,13 +228,7 @@ public String apply(String p1, String p2) {
                     "[A@0]:0+1+1"
             );
             proc2.checkAndClearProcessResult();
-            proc3.checkAndClearProcessResult(
-                    "[A@0]:null",
-                    "[B@0]:null",
-                    "[C@0]:null",
-                    "[D@0]:null",
-                    "[A@0]:null"
-            );
+            proc3.checkAndClearProcessResult();
```

Comment:
We do test inner KTable-KTable join and `null` values are ignored. (I cannot recall exactly: according to KIP, this should have been empty an result before this change -- however, it seem correct this way to me.)